cfme/tests/infrastructure/test_instance_analysis.py

```diff
@@ -9,51 +9,79 @@
 from cfme.common.provider import cleanup_vm
 from cfme.configure import configuration, tasks
 from cfme.infrastructure import host
-from cfme.infrastructure.pxe import get_template_from_config
 from cfme.provisioning import do_vm_provisioning
 from cfme.web_ui import InfoBlock, Table, SplitTable, paginator, tabstrip as tabs, toolbar as tb
 from fixtures.pytest_store import store
-from utils import testgen, ssh
+from utils import testgen, ssh, safe_string
+from utils.blockers import BZ
 from utils.conf import cfme_data
+from utils.log import logger
 from utils.wait import wait_for
 from utils.version import pick, LOWEST
 
+WINDOWS = {'id': "Red Hat Enterprise Windows", 'icon': 'linux_windows'}
+
+RPM_BASED = {
+    'rhel': {'id': "Red Hat", 'release-file': '/etc/redhat-release', 'icon': 'linux_redhat'},
```

Comment:
I'm tempted, "very" in fact, to suggest that we put the install and verify packaging commands here in these dicts for multiple reasons
1) It will greatly simplify the code near the bottom of the page for selecting the package name and install lines
2) It will allow us to potentially expand this section later with multiple OS versions, types and commands